src/lib/annotations/doc/docAnnotatorUtil.js

```diff
@@ -358,21 +358,32 @@ export function getContext(pageEl, annotationLayerClass, paddingTop, paddingBott
     }
 
     let annotationLayerEl = pageEl.querySelector(`.${annotationLayerClass}`);
+    let context;
     // Create annotation layer if one does not exist (e.g. first load or page resize)
     if (!annotationLayerEl) {
         annotationLayerEl = document.createElement('canvas');
         annotationLayerEl.classList.add(annotationLayerClass);
         const pageDimensions = pageEl.getBoundingClientRect();
         const pagePaddingTop = paddingTop || 0;
         const pagePaddingBottom = paddingBottom || 0;
-        annotationLayerEl.width = pageDimensions.width;
-        annotationLayerEl.height = pageDimensions.height - pagePaddingTop - pagePaddingBottom;
+        const pxRatio = window.devicePixelRatio || 1;
+        const width = pageDimensions.width;
+        const height = pageDimensions.height - pagePaddingTop - pagePaddingBottom;
+
+        annotationLayerEl.style.width = `${width}px`;
```

Comment:
The canvas height and width should be the ratio * display height and width. Meaning if the ratio is 2 we have 2 canvas pixels but the element appears to be only 1.